From 2db5f44adbca6f4aab6dfa17da80f1ad15eb1710 Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Mon, 13 Feb 2012 13:32:38 +0800
Subject: ENGR00174315 MX6Q max7310 set the default value of PCIE PWR ctrl2 to
 low

System would be halt, when the default value CTRL_2 is set to
high, change the default value to low.

root cause: System 3V3 would be dragged down to 1.5V for about 4ms.

Signed-off-by: Richard Zhu <r65037@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_arm2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_arm2.c b/arch/arm/mach-mx6/board-mx6q_arm2.c
index 17eb522..5ea5a2a 100644
--- a/arch/arm/mach-mx6/board-mx6q_arm2.c
+++ b/arch/arm/mach-mx6/board-mx6q_arm2.c
@@ -394,7 +394,7 @@ static int max7310_1_setup(struct i2c_client *client,
 	unsigned gpio_base, unsigned ngpio,
 	void *context)
 {
-	int max7310_gpio_value[] = { 0, 1, 1, 1, 0, 0, 0, 0 };
+	int max7310_gpio_value[] = { 0, 1, 0, 1, 0, 0, 0, 0 };
 
 	int n;
 
-- 
1.7.9.5

